app/views/projects/_form.html.erb

```diff
@@ -29,6 +29,10 @@
           class: 'govuk-label govuk-label--s'
         }
       %>
+      <%= f.govuk_text_area :related_links, rows: 4,
+        label: { text: 'Related content', size: 's' },
+        hint: { text: 'Use markdown for links',
+                class: 'govuk-hint govuk-!-font-size-16' } %>
```

Comment:
`govuk-!-font-size-16` means the hint text for this field is smaller than other hint text on the same page. Is this intended?

Also, might help to explain where this content appears.

Reply:
It wasn't intentional!